orchestrator/gravity_utils/src/connection_prep.rs

```diff
@@ -2,22 +2,26 @@
 //! It's a common problem to have conflicts between ipv4 and ipv6 localhost and this module is first and foremost supposed to resolve that problem
 //! by trying more than one thing to handle potentially misconfigured inputs.
```

Comment:
This file makes me slightly insane in how much it tries to auto recover from foot guns. We should definitely try and clean it up in a future PR

Reply:
Agreed.